Templates/ServiceBusTopicTrigger-NodeJS/function.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "mySbMsg",
+            "accessRights" : "Manage",
+            "type": "serviceBusTrigger",
+            "direction": "in",
+            "topicName": "myinputtopic",
```

Comment:
Followed the pattern for storage queue templates. Which doesn't apply to topic because a topic can have multiple subscribers. For queue i did not want 2 competing functions if the user simply clicked create without modifying the queue name.